### DISCUSSION TITLE
Amend a submission to SRBench 2022 Competition

### DISCUSSION THREAD

**Discussion by MiquelTriana** [cavalab/srbench#103] [Q&A] [answered]
Hi @lacava,

We recently submitted the method "QLattice" to the SRBench 2022 Competition, and it seems the pull request has been merged successfully. However, we noticed an error in the license of our library that we fixed in a new version (feyn==3.0.0 -> feyn==3.0.1). 
Would it be possible to amend our submission to use the new version (feyn==3.0.1)? Shall we do a new pull request with the fix?

Thank you for your time!

**Comment by lacava** [comment, marked as answer]
Sure, you can amend your submission until May 15.
